tests/fixtures/ntfs2ntfs/stops/stop_times.txt

```diff
@@ -19,6 +19,6 @@ DEFR,RERAB1,5,09:24:00,09:25:00,0,0,0,1,1,,,,2
 CDGR,RERAB1,8,09:39:00,09:40:00,0,0,0,0,0,,,,0
 GDLR,RERAB1,13,09:44:00,09:45:00,0,0,0,0,0,,,,0
 NATR,RERAB1,21,09:49:00,09:50:00,0,0,0,0,0,,,,0
-MTPZ,RERAB1,50,19:24:00,19:25:00,0,0,0,0,1,,,,2
+MTPZ,RERAB1,50,19:24:00,19:25:00,0,0,0,0,0,,,,0
```

Comment:
I'm not sure that I understand why this is changing?

Reply:
This change is due to another anomaly. Discussed with @prhod, this will be corrected later